crates/apollo-parser/test_data/parser/ok/0006_selection_with_fragment_spread.graphql

```diff
@@ -1,4 +1,7 @@
 {
-    pet
-    ...snackSelection
-}
\ No newline at end of file
+  pet
+  ...snackSelection
```

Comment:
since you're already editing these selection_with_fragment tests, could you also add the other fragment variants?

```suggestion
  ...snackSelection
  ... on Nap {
    cozyLocation
    durationOfNap
  }
  ...snackSelection @deprecated
  ... on Nap @provides(duration: "2 hours") {
    cozyLocation
  }
```

Reply:
ahhh oops just saw that you merged! i'll just quickly add these in another branch